### PR TITLE
Add com.github.cjbassi.ytop

### PR DESCRIPTION
**Pull request by tim77**
Initial manifest. [Expect](https://github.com/cjbassi/ytop/issues/63) upstream involvement here. The only thing which doesn't work: host processes not showing. This is understandable, but maybe we can fix this somehow or this is technically not possible?

### CODE REVIEW

File: com.github.cjbassi.ytop.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      - install -D -m 755 ./target/release/ytop -t /app/bin/
+      - install -D -m 644 ./com.github.cjbassi.ytop.appdata.xml -t /app/share/metainfo/
+      - install -D -m 644 ./com.github.cjbassi.ytop.svg -t /app/share/icons/hicolor/scalable/apps/
+      - install -D -m 644 ./com.github.cjbassi.ytop.desktop -t /app/share/applications/
+    sources:
+      - type: git
+        commit: d4eabcd415dbad33b24e31d3e89b0c79a893a355
+        url: https://github.com/cjbassi/ytop/
+      - type: file
+        path: data/com.github.cjbassi.ytop.appdata.xml
```
Comment (LeandroStanger) [inline com.github.cjbassi.ytop.yaml:30]: New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html

**Comment by LeandroStanger** [comment]
bot, build com.github.cjbassi.ytop

File: com.github.cjbassi.ytop.yaml (hunk 1 above)
Comment (tim77) [reply com.github.cjbassi.ytop.yaml:30]: Only for add-ons https://docs.fedoraproject.org/en-US/packaging-guidelines/AppData/
And appdata files installed in proper `/app/share/metainfo/`.

**Comment by tim77** [comment]
Failed on AppData validation. This is easy to fix.

```
• tag-missing           : <content_rating> required [use https://odrs.gnome.org/oars]
• tag-missing           : <release> required
• tag-missing           : <url> is not present
```

File: com.github.cjbassi.ytop.yaml (hunk 1 above)
Comment (bilelmoussaoui) [reply com.github.cjbassi.ytop.yaml:30]: I would follow appstream docs.

File: data/com.github.cjbassi.ytop.appdata.xml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>MIT</project_license>
+  
+  <description>
+    <p>
+      Another TUI based system monitor, this time in Rust!
+    </p>
+  </description>
+  
+  <categories>
```
Comment (bilelmoussaoui) [inline data/com.github.cjbassi.ytop.appdata.xml:18]: Should be defined in the desktop file. They are merged with appstream-compose later.

File: com.github.cjbassi.ytop.yaml (hunk 1 above)
Comment (tim77) [reply com.github.cjbassi.ytop.yaml:30]: OK, NP. Will fix this shortly.

**Comment by tim77** [comment]
bot, build com.github.cjbassi.ytop

**Comment by tim77** [comment]
bot, build com.github.cjbassi.ytop

**Comment by barthalion** [comment]
> The only thing which doesn't work: host processes not showing. This is understandable, but maybe we can fix this somehow or this is technically not possible?

ytop would need to execute a command on the host to list running processes, which implies upstream code changes.

**Comment by nedrichards** [comment]
bot, build com.github.cjbassi.ytop

**Comment by tim77** [comment]
bot, build com.github.cjbassi.ytop

**Pull request closed without merging**